yarn-project/circuit-types/src/l2_block.ts

```diff
@@ -146,7 +179,6 @@ export class L2Block {
       this.header.state.l1ToL2MessageTree,
       this.archive,
       this.body.getTxsEffectsHash(),
-      this.getL1ToL2MessagesHash(),
```

Comment:
Reverting this accidental change https://github.com/AztecProtocol/aztec-packages/pull/5371.